.github/workflows/devtools.yml

```diff
@@ -26,6 +26,9 @@ jobs:
 
     - name: Generate cache hash
       run: bash $GITHUB_WORKSPACE/lighthouse/.github/scripts/generate-devtools-hash.sh > cdt-test-hash.txt
+    # Caches are invalidated by GH after 7 days, so that's the max time between testing
```

Comment:
FWIW, this page says "GitHub will remove any cache entries that have not been accessed in over 7 days", which is slightly different.

https://docs.github.com/en/actions/advanced-guides/caching-dependencies-to-speed-up-workflows#usage-limits-and-eviction-policy

Reply:
good catch, let's just do this bit ourselves.